Libraries/LibWeb/Crypto/CryptoAlgorithms.cpp

```diff
@@ -3657,15 +3655,10 @@ WebIDL::ExceptionOr<JS::NonnullGCPtr<JS::Object>> HMAC::export_key(Bindings::Key
         }
 
         // Set the key_ops attribute of jwk to equal the usages attribute of key.
-        auto key_usages = verify_cast<JS::Array>(key->usages());
-        auto key_usages_length = MUST(MUST(key_usages->get(vm.names.length)).to_length(vm));
-        for (auto i = 0u; i < key_usages_length; ++i) {
-            auto usage = key_usages->get(i);
-            if (!usage.has_value())
-                break;
-
-            auto usage_string = TRY(usage.value().to_string(vm));
-            jwk.key_ops->append(usage_string);
+        jwk.key_ops = Vector<String> {};
+        jwk.key_ops->ensure_capacity(key->internal_usages().size());
+        for (auto const& usage : key->internal_usages()) {
+            jwk.key_ops->append(Bindings::idl_enum_to_string(usage));
```

Comment:
This makes more sense, thanks! The code was partially copied from another function in this file, that should probably be updated as well.

Reply:
The only difference is that this used to append directly to the optional in the struct, and that is null at first.
The other example sets it to an empty Vector first, so theres no crash.

But this way is a lot cleaner, so let me update that too :D